site/pages/disruption-detail/[disruptionId].page.tsx

```diff
@@ -602,11 +602,19 @@ const DisruptionDetail = ({
                         </div>
                         <Link
                             role="button"
-                            href={{
-                                pathname: `${TYPE_OF_CONSEQUENCE_PAGE_PATH}/${disruption.disruptionId}/${nextIndex}`,
-                                query: { return: DISRUPTION_DETAIL_PAGE_PATH },
-                            }}
-                            className="govuk-button mt-2 govuk-button--secondary"
+                            href={
```

Comment:
We should probably also impose this restriction at the zod schema level

Reply:
done